package androidx.compose.ui.text.font;

import androidx.compose.runtime.a3;
import kotlin.jvm.internal.DefaultConstructorMarker;

/* loaded from: classes.dex */
public interface e0 extends a3 {

    /* loaded from: classes.dex */
    public static final class a implements e0 {

        /* renamed from: a, reason: collision with root package name */
        private final Object f6197a;

        /* renamed from: b, reason: collision with root package name */
        private final boolean f6198b;

        public a(Object obj, boolean z10) {
            this.f6197a = obj;
            this.f6198b = z10;
        }

        public /* synthetic */ a(Object obj, boolean z10, int i11, DefaultConstructorMarker defaultConstructorMarker) {
            this(obj, (i11 & 2) != 0 ? true : z10);
        }

        @Override // androidx.compose.ui.text.font.e0
        public boolean a() {
            return this.f6198b;
        }

        @Override // androidx.compose.runtime.a3
        public Object getValue() {
            return this.f6197a;
        }
    }

    boolean a();
}
